src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1319,30 +1319,42 @@ export class HtmlVideoPlayer {
     /**
          * @private
          */
-    requiresCustomSubtitlesElement() {
-        // after a system update, ps4 isn't showing anything when creating a track element dynamically
-        // going to have to do it ourselves
-        if (browser.ps4) {
-            return true;
-        }
+    requiresCustomSubtitlesElement(userSettings) {
+        const subtitleAppearance = userSettings.getSubtitleAppearanceSettings();
+        switch (subtitleAppearance.subtitleStyling) {
+            case 'Native':
+                return false;
+            case 'Custom':
+                return true;
+            default:
+                // after a system update, ps4 isn't showing anything when creating a track element dynamically
+                // going to have to do it ourselves
+                if (browser.ps4) {
+                    return true;
+                }
 
-        if (browser.web0s) {
-            return true;
-        }
+                if (browser.web0s) {
+                    return true;
+                }
 
-        if (browser.edge) {
-            return true;
-        }
+                if (browser.edge) {
+                    return true;
+                }
+                // font-size styling does not seem to work natively in firefox. Switching to custom subtitles element for firefox.
```

Comment:
Blank line for consistent style.
```suggestion

                // font-size styling does not seem to work natively in firefox. Switching to custom subtitles element for firefox.
```